src/demo/demoUtils.ts

```diff
@@ -1,14 +1,69 @@
+import { LocaleConfig, SupportedLanguages } from '~types/locales'
+import { StepConfig, StepTypes } from '~types/steps'
+import { ServerRegions, SdkOptions } from '~types/sdk'
+
+type StringifiedBoolean = 'true' | 'false'
+
+export type QueryParams = {
+  countryCode?: StringifiedBoolean
+  disableAnalytics?: StringifiedBoolean
+  forceCrossDevice?: StringifiedBoolean
+  hideOnfidoLogo?: StringifiedBoolean
+  language?: 'customTranslations' | SupportedLanguages
+  link_id?: string
+  liveness?: StringifiedBoolean
+  multiDocWithInvalidPresetCountry?: StringifiedBoolean
+  multiDocWithPresetCountry?: StringifiedBoolean
+  noCompleteStep?: StringifiedBoolean
+  oneDoc?: StepTypes
+  oneDocWithCountrySelection?: StringifiedBoolean
+  oneDocWithPresetCountry?: StringifiedBoolean
+  poa?: StringifiedBoolean
+  region?: string
+  shouldCloseOnOverlayClick?: StringifiedBoolean
+  showCobrand?: StringifiedBoolean
+  smsNumber?: StringifiedBoolean
+  snapshotInterva?: StringifiedBoolean
+  snapshotInterval?: StringifiedBoolean
+  uploadFallback?: StringifiedBoolean
+  useHistory?: StringifiedBoolean
+  useLiveDocumentCapture?: StringifiedBoolean
+  useMemoryHistory?: StringifiedBoolean
+  useModal?: StringifiedBoolean
+  useMultipleSelfieCapture?: StringifiedBoolean
+  useUploader?: StringifiedBoolean
+  useWebcam?: StringifiedBoolean
+}
+
+export type CheckData = {
+  applicantId: string | null
+  sdkFlowCompleted: boolean
+}
+
+export type ViewOptions = {
```

Comment:
I think we should start to move away from the word "option" as it can be vague and we use it pretty much everywhere. How about `UIConfigurations`?

Reply:
Yeah makes sense, but I prefer `UIConfigs` for a more concise name

Reply:
Yes, I agree